week3_model_free/experience_replay.ipynb

```diff
@@ -83,12 +83,12 @@
     "        Parameters\n",
     "        ----------\n",
     "        size: int\n",
-    "            Max number of transitions to store in the buffer. When the buffer\n",
-    "            overflows the old memories are dropped.\n",
+    "            Max number of transitions to store in the buffer. When the buffer is\n",
+    "            overflowed, the old memories are dropped.\n",
```

Comment:
I think "overflows" is fine here, since this phrase is quite colloquial. Haven't googled for confirmation though.